store/store.go

```diff
@@ -37,16 +49,6 @@ import (
 	"strings"
 	"sync"
 	"time"
-
-	"github.com/snapcore/snapd/arch"
-	"github.com/snapcore/snapd/asserts"
-	"github.com/snapcore/snapd/dirs"
-	"github.com/snapcore/snapd/logger"
-	"github.com/snapcore/snapd/osutil"
-	"github.com/snapcore/snapd/overlord/auth"
-	"github.com/snapcore/snapd/progress"
-	"github.com/snapcore/snapd/release"
-	"github.com/snapcore/snapd/snap"
```

Comment:
Please don't do this; the convention we're using for imports is to have three groups: standard library, others, and snapd. Keeps things sane.